### PR TITLE
Document other bind options on --volumes flag

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@mheon @TomSweeneyRedHat @baude @jwhonce @giuseppe @QiWang19 PTAL

**Comment by QiWang19** [comment]
LGTM

File: docs/source/markdown/podman-create.1.md [hunk 1]
```diff
@@ -817,6 +817,10 @@ container. The `OPTIONS` are a comma delimited list and can be:
 * [rw|ro]
 * [z|Z]
 * [`[r]shared`|`[r]slave`|`[r]private`]
+* [`[r]bind`]
+* [`exec`|`noexec`]
```
Comment (mheon) [inline docs/source/markdown/podman-create.1.md]: Should this swap to `noexec | exec` to match the other two?

File: docs/source/markdown/podman-create.1.md [hunk 2]
```diff
@@ -870,6 +874,23 @@ where source dir is mounted on) has to have right propagation properties. For
 shared volumes, source mount point has to be shared. And for slave volumes,
 source mount has to be either shared or slave.
 
+If you want to recursively mount a volume and all of it's submounts into a
+container, then you can use the `rbind` option.  By default --volumes use the
```
Comment (mheon) [inline docs/source/markdown/podman-create.1.md]: Suggest "By default the `bind` option is used, and submounts of the source directory will not be mounted into the container"

File: docs/source/markdown/podman-create.1.md [hunk 3]
```diff
@@ -870,6 +874,23 @@ where source dir is mounted on) has to have right propagation properties. For
...
+in the container.
+
+Mounting the volume with the `nosuid` options means that SUID applications on
+the volume will not be able to raise privilege.
+
+Mounting the volume with the noexec option means that no executables on the
+volume will be able to executed within the container.
+
+Mounting the volume with the nodev option means that no devices on the volume
+will be able to be used by processes within the container.
```
Comment (mheon) [inline docs/source/markdown/podman-create.1.md]: This is default, should mention that

File: docs/source/markdown/podman-create.1.md [hunk 4]
```diff
@@ -870,6 +874,23 @@ where source dir is mounted on) has to have right propagation properties. For
 shared volumes, source mount point has to be shared. And for slave volumes,
 source mount has to be either shared or slave.
 
+If you want to recursively mount a volume and all of it's submounts into a
+container, then you can use the `rbind` option.  By default --volumes use the
+`bind` option.  Any submounts of the <source-dir> will not get mounted
+in the container.
+
+Mounting the volume with the `nosuid` options means that SUID applications on
+the volume will not be able to raise privilege.
```
Comment (mheon) [inline docs/source/markdown/podman-create.1.md]: This is default, should mention that

File: docs/source/markdown/podman-create.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/source/markdown/podman-create.1.md]: Fixed

File: docs/source/markdown/podman-create.1.md (hunk 2 above)
Comment (rhatdan) [reply docs/source/markdown/podman-create.1.md]: Fixed

File: docs/source/markdown/podman-create.1.md (hunk 3 above)
Comment (rhatdan) [reply docs/source/markdown/podman-create.1.md]: Fixed

File: docs/source/markdown/podman-create.1.md (hunk 4 above)
Comment (rhatdan) [reply docs/source/markdown/podman-create.1.md]: Fixed

File: docs/source/markdown/podman-run.1.md [hunk 5]
```diff
@@ -913,6 +917,25 @@ where source dir is mounted on) has to have right propagation properties. For
 shared volumes, source mount point has to be shared. And for slave volumes,
 source mount has to be either shared or slave.
 
+If you want to recursively mount a volume and all of it's submounts into a
+container, then you can use the `rbind` option.  By default the bind option is
+used, and submounts of the source directory will not be mounted into the
+container.
+
+Mounting the volume with the `nosuid` options means that SUID applications on
+the volume will not be able to raise privilege. By default volumes
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-run.1.md]: nit picking.  Might just be me, but I'm not feeling "raise privilege".  Perhaps "change their privilege".

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @mheon @baude @vrothberg @giuseppe @QiWang19 @jwhonce PTAL
Lets get this merged...

**Comment by mheon** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by stevekuznetsov** [comment]
FYI this was thrashing...

```json
{
  "org": "containers",
  "repo": "libpod",
  "branch": "master",
  "pr": 4557,
  "msg": "Merge failed.",
  "level": "error",
  "error": "PR is unmergable. Do the Tide merge requirements match the GitHub settings for the repo? Required status check \"success\" is queued.",
  "time": "2019-11-26T17:47:27Z"
}
```

**Pull request merged**